elements/media-quote/src/media-quote.js

```diff
@@ -23,6 +24,241 @@ class MediaQuote extends LitElement {
+            }
+
+          figure {
+            width: 60%;
+            z-index: 0;
+          }
+
+          img {
+            width: 100%;
+            z-index: 1;
+          }
+
+          .caption {
+            border: var(--ddd-border-lg);
+            border-color: var(--ddd-theme-primary, var(--ddd-primary-1));
+          }
+
+          .controller:hover {
+            cursor: pointer;
+          }
+
+          .controller {
+            background-color: var(--ddd-theme-primary, var(--ddd-primary-1));
+            color: var(--lowContrast-override, var(--ddd-theme-bgContrast));
+            display: flex;
+            align-items: center;
+            padding: var(--ddd-spacing-0) var(--ddd-spacing-2);
+          }
+
+          .controller-text {
+            display: flex;
+            justify-content: space-between;
+            width: 100%;
+          }
+
+          .controller:hover .show-hide, .controller:focus .show-hide {
+            text-decoration: underline;
+          }
+
+          .caption-content {
+            display: none;
+          }
+
+          figcaption {
+            color: var(--ddd-theme-default-potentialMidnight); 
+            width: 98%;
+            padding: var(--ddd-spacing-2);
+          }
+
+          @container (max-width: 1261px) and (min-width: 1000px) {
+            .quote {
+              font-size: var(--ddd-font-size-xs);
+            }
+
+            .author, .author-detail {
+              font-size: var(--ddd-font-size-xxs);
+            }
+          }
+
+          @container (max-width: 999px) { /* Mobile devices */
+            .text-overlay {
+              position: relative;
+              width: 90%;
+              top: 0%;
+              left: 0%;
+              font-size: var(--ddd-font-size-xs);
+            }
+
+            .text-overlay {
+              text-align: center;
+            }
+
+            .author, .author-detail {
+              font-size: var(--ddd-font-size-xxs);
+            }
+
+            figure {
+              width: 100%;
+            }
+
+            .content {
+              box-decoration-break: unset;
+              display: block;
+            }
+
+            .content, .citation {
+              padding: var(--ddd-spacing-2) var(--ddd-spacing-2);
+            }
+
+            figcaption {
+              width: 95%;
+            }
+          }
+      `,
+    ];
+  }
+
+  render() {
+    return html`
+        <div class="media-quote-container">
+          <figure>
+            <div class="top-content">
+              <div class="text-overlay"> 
+                <p class="quote">
+                  <span class="content"><slot name="quote"></slot></span>
+                    ${this.querySelector('[slot="author"]') && this.querySelector('[slot="author"]').textContent.trim().length > 0 ? html`
+                      <span class="citation">
+                        <span class="author">- <slot name="author"></slot></span> 
```

Comment:
Make a field for author as well like `<slot name="author">${this.author}</slot>`

Reply:
I assume I would need to define an author variable in `constructor()` and `properties()`, and do something similar with other slots?